test/e2e/specs/editor/plugins/wp-editor-meta-box.spec.js

```diff
@@ -0,0 +1,56 @@
+/**
+ * WordPress dependencies
+ */
+const { test, expect } = require( '@wordpress/e2e-test-utils-playwright' );
+
+test.describe( 'WP Editor Meta Boxes', () => {
+	test.beforeAll( async ( { requestUtils } ) => {
+		await requestUtils.activatePlugin(
+			'gutenberg-test-plugin-wp-editor-meta-box'
+		);
+	} );
+
+	test.afterAll( async ( { requestUtils } ) => {
+		await requestUtils.deactivatePlugin(
+			'gutenberg-test-plugin-wp-editor-meta-box'
+		);
+	} );
+
+	test( 'Should save the changes', async ( { admin, editor, page } ) => {
+		await admin.createNewPost();
+		// Add title to enable valid non-empty post save.
+		await page.type( '.editor-post-title__input', 'Hello Meta' );
```

Comment:
It'd be good to change this to a role selector:
```suggestion
		await page.type( 'role=textbox[name="Add title"i]', 'Hello Meta' );
```